src/Symfony/Component/Form/Button.php

```diff
@@ -371,8 +371,8 @@ public function handleRequest($request = null)
     /**
      * Submits data to the button.
      *
-     * @param string|null $submittedData Not used
-     * @param bool        $clearMissing  Not used
+     * @param array|string|null $submittedData Not used
```

Comment:
I am not sure how the data can end up to be an array here for a button.

Reply:
This signature is a requirement brought by FormInterface